crates/cargo-contract/src/cmd/config.rs

```diff
@@ -74,18 +71,17 @@ impl Environment for Ecdsachain {
 
 impl SignerConfig<Self> for Ecdsachain
 where
-    <Self as Config>::Signature: From<sp_core::ecdsa::Signature>,
+    <Self as Config>::Signature: From<MultiSignature>,
```

Comment:
No longer the `substrate-compat` feature, so we use the `subxt` defined types by default.